dissect-netwayste/src/netwaysteparser.rs

```diff
@@ -108,32 +108,31 @@ fn parse_size_from_type(type_arg: String) -> Vec<Sizing> {
         }
 
         // Consume characters until we reach the inner type
-        let mut characters = param.chars();
```

Comment:
This is really a comment about the first line in this function: if `param` is what you want to call this, you could just have that in the function arguments, then there's no need for `let param = type_arg;`

Reply:
Ah thanks! I think initially I wanted to preserve the `type_arg` for some reason but I moved away from it.